sql/core/src/test/scala/org/apache/spark/sql/TestQueryExecutionListener.scala

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Is it possible to modify this slightly and reuse it?  https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/util/ExecutionListenerManagerSuite.scala#L48

Reply:
I think it's possible. Just took a look; however, mind if I had a separate one as is for Python test specifically? maybe I am too much worried but thinking about having a dependency with a class in a suite and I am a bit hesitant.

Reply:
Yeah, I think that's fine. Thanks for putting a comment in the class for what it is for.